scripts/init-project.sh

```
@@ -1,65 +1,85 @@
+# - Eliminar sección de builder completo
+
+sed -i '/name: 🔨 Build & Push BUILDER image/,/name: ⚙️ Build & Push RUNTIME image/{//!d}' "\$WORKFLOW"
 
-#  - Renombrar la cabecera de runtime
-sed -i "s|name: ⚙️ Build & Push RUNTIME image|name: 🔨 Build & Push ${PROJECT_NAME}-runtime image|g" "$WORKFLOW"
+# - Renombrar la cabecera runtime
 
-#  - Actualizar comandos Docker para runtime
-sed -i -E "s|docker build --target production.*|docker build --target production \\\n            -t $DOCKER_USER/${PROJECT_NAME}-runtime:\${{ env.VERSION }} \\\n            -t $DOCKER_USER/${PROJECT_NAME}-runtime:latest .|g" "$WORKFLOW"
-sed -i -E "s|docker push .+|docker push $DOCKER_USER/${PROJECT_NAME}-runtime:\${{ env.VERSION }} \\\n            docker push $DOCKER_USER/${PROJECT_NAME}-runtime:latest|g" "$WORKFLOW"
+UPPER\_NAME=\$(printf '%s' "\$PROJECT\_NAME" | tr '\[:lower:]' '\[:upper:]')
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)